README.md

```
@@ -140,6 +140,91 @@ openclaw honcho ask <question>                  # Query Honcho about the user
+    "qmd": {
+      "limits": {
+        "timeoutMs": 120000
+      }
+    }
+  }
+}
+```
+
+The default timeout is 4000ms which is too short for most QMD operations. Setting it to 120000ms (2 minutes) gives QMD enough time. You can verify it's working in the logs:
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)